### PR TITLE
Bring back support for VnetId and SubnetName

### PR DESCRIPTION
**Pull request by solankisamir**
Regenerate operation with Support for VnetId and Subnetname

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@solankisamir__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sasolank). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by solankisamir** [comment]
@cormacpayne  can you take a look at the PR which brings back support for just getting the properties?

File: src/ResourceManagement/ApiManagement/ApiManagementManagement/Microsoft.Azure.Management.ApiManagement.nuget.proj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Management.ApiManagement
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Management.ApiManagement">
-      <PackageVersion>3.0.0-preview</PackageVersion>
+      <PackageVersion>3.0.1-preview</PackageVersion>
```
Comment (markcowl) [inline src/ResourceManagement/ApiManagement/ApiManagementManagement/Microsoft.Azure.Management.ApiManagement.nuget.proj]: This needs to be be 3.1.0, as there is new api surface

**Review by markcowl — CHANGES_REQUESTED** [review]
Also need test coverage for the new model - need at least one test that exercises the new api surface

File: src/ResourceManagement/ApiManagement/ApiManagementManagement/Microsoft.Azure.Management.ApiManagement.nuget.proj (hunk 1 above)
Comment (solankisamir) [inline src/ResourceManagement/ApiManagement/ApiManagementManagement/Microsoft.Azure.Management.ApiManagement.nuget.proj]: Updated the nuget to 3.1.0 and added validation on the entities. We cannot create a test, as we do not allow set on the two properties. Only services with existing setup can view the properties (subnetname and VnetId).

**Comment by solankisamir** [comment]
@markcowl can you take a look?

**Review by markcowl — APPROVED** [review]

**Pull request merged**